api/src/main/java/org/pmiops/workbench/db/dao/UserDao.java

```diff
@@ -148,22 +148,37 @@
     String getAccessTierShortNames();
   }
 
+  /** Important! Make sure add alias and it matches name in {@link DbAdminTableUser} */
```

Comment:
Thanks for adding this!  Could you add a little more about why, to guide future maintainers?  Specifically the fact that this will cause null results if incorrect.